### DISCUSSION TITLE
sleep_mode brightness not set?

### DISCUSSION THREAD

**Discussion by Elkropac** [basnijholt/adaptive-lighting#453] [Q&A]
Hi,
i use adaptive lightning for lamps in bedroom. It is set to white mode and alter brightness in range from 48% to 35%.
I also have sleep mode set to rgb_color (amber) and 50% brightness.

I have media player here. When it starts playback, it disables "brightness adapt" and sets brightness to 20% manually. When playback stops or pauses, it enables brightness adapt back.
Everything works great so far, brightness goes up.

At 20:30, i have automation, which enables sleep_mode. When media player is in play mode, it just changes the color of light, brighness stays on 20%. But when playback stops and "brightness adapt" is enabled again, it does not change to 50%.

Is that expected?

**Comment by th3w1zard1** [comment]
This should be normal, adaptive_lighting only adapts lights every interval so you might not be seeing a instantaneous brightness change.

You can be safe and toggle off and on the main switch in your automation to resume normal adapting if you like.

**Comment by Elkropac** [comment]
Hi i will try it. 
But when I'm not in sleep_mode, brightness goes to computed value at the moment i enable "brightness adapt" switch.
In sleep_mode, it does not go to brightness specified by sleep_mode

**Reply by th3w1zard1** [reply]
You need to post more info otherwise we have no idea how to help you.
Post your logs, automations, scripts, your adaptive lighting config and the version of adaptive lighting you're running (1.7.0 is the latest)

**Reply by Elkropac** [reply]
Thanks for response, i was unable to test it yet. I only noticed, that brightness does not return to computed value even in non-sleep mode. I will try with version 1.7.0 soon
